platform/src/main/java/org/stellar/anchor/platform/data/JdbcSep24Transaction.java

```diff
@@ -30,15 +30,19 @@ public String getProtocol() {
   String kind;
 
   @SerializedName("status_eta")
+  @Column(name = "status_eta")
```

Comment:
Are all this `@Column` annotations necessary?

Reply:
It is not necessary. But it is to lock the column name caused by the inconsistency between tools and Spring. 

There are pros and cons of specifying the column name. I personally don't like to specify the column name just because of a bug and being paranoid of tool bugs. 

WDYT?

Reply:
I'd prefer to only do it when necessary, otherwise it's too verbose in my opinion

Reply:
I'm confused, isn't using `@Column` the only way to ensure Spring uses the correct DB column name? Are there alternatives?

Reply:
Spring has an automatic conversion that's been used before. I guess in rare cases it doesn't match to expected naming

Reply:
Ok, then lets lock in the column name. Not ideal but sounds necessary.

Reply:
locking is verbose but safe. We do not expect too many Entity classes and they are quite stable. I think we can lock it unless we have other strong reasons in the future.